documentation/docs/libraries/rust/how_to/_09_get_block/_key_api_ref_links.md

```diff
@@ -0,0 +1,19 @@
+### `Client.get_block(blockId)`
+
+You can retrieve a block's data using its block ID using
+the [`Client. get_block_metadata(blockId)`](./../libraries/nodejs/references/classes/Client#getblock)
```

Comment:
Shouldn't be metadata, and the prototype is wrong ? Should be `get_block(&self, block_id: &BlockId)` ?

Reply:
```suggestion
the [`Client.get_block(&self, block_id: &BlockId)`](./../libraries/nodejs/references/classes/Client#getblock)
```